### ISSUE TITLE
ES still active?

### ISSUE THREAD

**Issue by ChristianWilkie** [nooperation/d2es-stuff#5]
hey, I played D2 ES ages ago on the SA forums and had a lot of fun, starred this repo and totally forgot about it but recently saw some updates to it

is ES still active? and is it going to be added w/ the D2 remake thing?

**Comment by nooperation** [comment]
Yeah, they seem to have another server up and running again with a few playing it. They've modded it quite a bit since the old days so I'm just updating and tweaking a few things along the way to try and lessen the amount of carpal tunnel from constantly clicking things

I'm not really sure of the state of ES in general though. I know the main site for it appears to have been taken down unfortunately. I'd love it if the D2 remake supports this kind of modding, that would be amazing
